### PR TITLE
[Commit for models]

### PR DESCRIPTION
**Pull request by ArpithaS9**
Created a folder 

Models
1. Tasks.ts
2. User.ts

### CODE REVIEW

File: src/app/models/Task.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export class Task{
+    public uId:number;
+    public title:string;
+    public description:string;
+    public status:string;
+    public priority:string;
+    public assignedTo:any;
```
Comment (balaji01Sawant) [inline src/app/models/Task.ts]: Why are we using "any" to "assignedTo"? Can we use specific data type?

File: src/app/models/User.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    public uId:number;
+    public firstName:string;
+    public lastName:number;
+    public email:string;
+    public roles:string;
+
+    constructor(id:number,fName:string,lName:number,email:string,roles:string)
+    {
+        this.uId=id;
+        this.firstName=fName;
```
Comment (balaji01Sawant) [inline src/app/models/User.ts]: this.firstName=firstName;

File: src/app/models/User.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        this.lastName= lName;
```
Comment (balaji01Sawant) [inline src/app/models/User.ts]: this.lastName=lastName;

File: src/app/models/Task.ts (hunk 1 above)
Comment (ArpithaS9) [reply src/app/models/Task.ts]: Sure will declare with specific datatype.

**Comment by ArpithaS9** [comment]
Need to modify the code as per the comments

File: src/app/models/Task.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+export class Task{
+    public uId:number;
```
Comment (Ashwini-Wellsky) [inline src/app/models/Task.ts:2]: We can use more descriptive names like userId

File: src/app/models/Task.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    public title:string;
+    public description:string;
+    public status:string;
+    public priority:string;
+    public assignedTo:number;
+    public createdAt:any;
+    public updatedAt:any;
```
Comment (Ashwini-Wellsky) [inline src/app/models/Task.ts:9]: Do we also need modifiedBy?

File: src/app/models/User.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+export class User{
+    public uId:number;
+    public firstName:string;
+    public lastName:number;
+    public email:string;
+    public roles:string;
```
Comment (Ashwini-Wellsky) [inline src/app/models/User.ts:6]: This looks good too but try to have an enum for Roles.

**Pull request closed without merging**